From 9eab4e39d83429c90457dc2063c45797bf9c9409 Mon Sep 17 00:00:00 2001
From: Ookiineko <chiisaineko@protonmail.com>
Date: Thu, 18 Aug 2022 17:40:38 +0800
Subject: [PATCH 1/2] CMakeLists: silence find_program() warns

Signed-off-by: Ookiineko <chiisaineko@protonmail.com>
---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 35ff4c11..2adcfcc3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,7 @@
 cmake_minimum_required(VERSION 3.5)
+if(POLICY CMP0109)
+  cmake_policy(SET CMP0109 OLD)
+endif()
 
 # Defer enabling C and CXX languages.
 project(BoringSSL NONE)
-- 
2.38.0

